exercises/Exercise-2-datasets-and-collections-of-data.ipynb

```diff
@@ -231,37 +259,47 @@
     "\n",
     "<div id=\"solution6\" class=\"collapse\">\n",
     "<br>\n",
-    "<code>diamond_ds.to(hv.BoxWhisker, 'cut', 'price', groupby='clarity')</code>\n",
+    "<code>hm = diamond_ds.to(hv.BoxWhisker, 'cut', 'price', groupby='clarity')\n",
+    "hm</code>\n",
     "</div>\n",
     "\n",
-    "This time let's lay out the grouped dimension as a grid. Use the ``grid`` method on the grouped dataset. Then enable the ``shared_xaxis`` and ``shared_yaxis`` plot options on the resulting ``GridSpace``. Finally set a custom ``xrotation`` to rotate the x-axis ticks.\n",
+    "This time let's lay out the grouped dimension as a grid. Use the ``grid`` method on the grouped dataset. Then enable the ``shared_xaxis`` and ``shared_yaxis`` plot options on the resulting ``GridSpace``. Finally set a custom ``xrotation`` to rotate the x-axis ticks, being careful to rotate it on the correct element type.\n",
     "\n",
-    "<b><a href=\"#hint4\" data-toggle=\"collapse\">Hint</a></b>\n",
+    "<b><a href=\"#hint7\" data-toggle=\"collapse\">Hint</a></b>\n",
     "\n",
-    "<div id=\"hint4\" class=\"collapse\">\n",
-    "Just like elements, containers like a ``GridSpace`` have plot options which can be specified using ``%%opts GridSpace [...].\n",
+    "<div id=\"hint7\" class=\"collapse\">\n",
+    "Just like elements, containers like a ``GridSpace`` have plot options which can be specified using ``%%opts GridSpace [...]``. But the ``xrotation`` won't need to be applied to the ``GridSpace``, but to the underlying element in each grid cell.\n",
     "</div>"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": []
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<b><a href=\"#solution6\" data-toggle=\"collapse\">Solution</a></b>\n",
+    "<b><a href=\"#solution7\" data-toggle=\"collapse\">Solution</a></b>\n",
```

Comment:
Something is off here, the link here must match the id below.

Reply:
Thanks. There were several in this file that were mixed up like that previously, so I had to renumber things overall, but it looks like I needed to renumber one more. Fixed now.